author.php

```diff
@@ -0,0 +1,29 @@
+<?php get_header(); ?>
+<?php get_template_part('breadcrumb'); ?>
+<?php get_template_part( 'header-blog' ); ?>
+
+<div id="primary" class="content-area author-page">
+    <div class="container">
+        <div class="row">
+            <main id="main" class="col-xs-12 col-sm-8 col-md-8" role="main">
+                <article>
+                    <div class="entry-header">
+                        <h2>
+                            <?php
+                            $curauth = (isset($_GET['author_name'])) ? get_user_by('slug', $author_name) : get_userdata(intval($author));
```

Comment:
Looking at this block of code I will possible think of:
1. Sanitize $_GET["author_name"] using something like e.g. sanitize_text_field($_GET["author_name"])
2. Adding it into the function and pass the $_GET variable as a parameter. By doing that you will end up having only functions inside this PHP file and unit test.

Reply:
This is only checking if the $_GET exists. No need to sanitize it as it's not outputted in anyway.